squid-mixin/README.md

```diff
@@ -0,0 +1,101 @@
+# Squid Mixin
+
+Squid mixin is a set of configurable Grafana dashboards and alerts.
+
+The Squid mixin contains the following dashboards:
+
+- Squid overview
+
+and the following alerts:
+
+- SquidHighPercentageOfHTTPServerRequestErrors
+- SquidHighPercentageOfFTPServerRequestErrors
+- SquidHighPercentageOfOtherServerRequestErrors
+- SquidHighPercentageOfClientRequestErrors
+- SquidLowCacheHitRatio
+
+## Squid Overview
+
+The Squid overview dashboard provides details on both server and client HTTP, FTP, and other requests, caching hits and misses, and both cache and access logs.
+
+![TODO Dashboard screenshots]()
+
+```
+{
+  _config+:: {
+    enableLokiLogs: false,
+  },
+}
+```
+
+In order for the selectors to properly work for access and cache logs ingested into your logs datasource, please also include the matching `job` label onto the [scrape_configs](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#scrape_configs) as to match the labels for ingested metrics.
```

Comment:
you call out the job label, but you should also call out the instance label (at least I think since you are setting it here in your example config)

Reply:
👍